src/project.godot

```diff
@@ -115,6 +115,7 @@ custom_default_config="res://default_panku_config.cfg"
 
 [physics]
 
+common/physics_ticks_per_second=10
```

Comment:
I .. don't think we want to commit to this?

Reply:
oops, I've been using it for so long I forgot I changed it 😂 it shouldn't matter since we turned interpolation on but let me remove it, good catch